lib/OnyxUtils.ts

```diff
@@ -256,11 +258,18 @@ function getAllKeys(): Promise<Set<OnyxKey>> {
 }
 
 /**
- * Checks to see if the a subscriber's supplied key
+ * @returns {Set} A set of collection keys.
+ */
+function getCollectionKeys() {
```

Comment:
Don't forget to add return types to every function, we could also enforce this with an eslint rule